packages/components/package.json

```diff
@@ -19,6 +19,7 @@
   },
   "dependencies": {
     "@tektoncd/dashboard-utils": "file:../utils",
+    "classnames": "^2.3.1",
```

Comment:
This is only used within components and graphs packages. So we could remove it from the main package.json? https://github.com/tektoncd/dashboard/blob/main/package.json#L31

Reply:
Absolutely, I thought I had done that with this change but clearly not. Good catch!

Reply:
Updated